### ISSUE TITLE
rest-api-cloud-client test FAIL

### ISSUE THREAD

**Issue by vasap87** [StnetixDevTeam/ariADDna#79]
```
mvn clean install
```
on module rest-api-cloud-client fail on test:
```test
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running com.stnetix.ariaddna.externalcloudapi.test.IAbstractCloudTest
Directories 
        Local Storage
        Local Storage/MyDocs
        Local Storage/MyDocs/My Books
        Local Storage/MyDocs/My Books/Картинки created
Downloading files...
Done
CTNB001154/127.0.1.1
Files were deleted
Directories were deleted
Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 8.458 sec <<< FAILURE! - in com.stnetix.ariaddna.externalcloudapi.test.IAbstractCloudTest
inet(com.stnetix.ariaddna.externalcloudapi.test.IAbstractCloudTest)  Time elapsed: 0.106 sec  <<< ERROR!
org.mockito.exceptions.misusing.MissingMethodInvocationException: 
when() requires an argument which has to be 'a method call on a mock'.
For example:
    when(mock.getArticles()).thenReturn(articles);

Also, this error might show up because:
1. you stub either of: final/private/equals()/hashCode() methods.
   Those methods *cannot* be stubbed/verified.
   Mocking methods declared on non-public parent classes is not supported.
2. inside when() you don't call method on mock but on some other object.

        at com.stnetix.ariaddna.externalcloudapi.test.IAbstractCloudTest.inet(IAbstractCloudTest.java:142)


Results :

Tests in error: 
  IAbstractCloudTest.inet:142 MissingMethodInvocation 
when() requires an argume...

Tests run: 2, Failures: 0, Errors: 1, Skipped: 0

```

**Comment by ancord-m** [comment]
The problem is a bit complicated. Mockito was used in a wrong way. The unit-test became integration test...  I guess this issue should be closed. I need to re-write the test completely and make changes in IAbstractCloud interface.

**Comment by vasap87** [comment]
Fix in #84
